sw/device/tests/dif/dif_clkmgr_unittest.cc

```diff
@@ -0,0 +1,293 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_clkmgr.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/testing/mock_mmio.h"
+
+#include "clkmgr_regs.h"
```

Comment:
Should include `// Generated.` comment, since it's a generated header.

Reply:
Done